meson.build

```diff
@@ -29,6 +29,7 @@ endif
 
 if get_option('with_analyzer')
   add_project_arguments('-fanalyzer', language : 'c')
+  add_project_arguments('-Wno-analyzer-malloc-leak', language : 'c')
```

Comment:
`-fanalyzer` finds a few possible malloc leaks. They are quite complex to track down/understand and need verification that they are actually leaks or maybe false-positives. Because of that I turned this option off for now and tackle the malloc leaks in dedicated issues. This way we already have the static analysis in the CI and prevent other issues.